pulumi/github/repos/core_repos.yml

```diff
@@ -0,0 +1,16 @@
+- .github
```

Comment:
Rather than tracking these here, can we share the info from the website repo so that we don't have to track it twice in two locations?

The website repo has a public `pipelines.json` file that lists all valid pipeline repos. The "repos not to track" are in an internal file but it'd be easy to make that public.

Reply:
Perfect, I just copied in whatever was on the check_health to get the "core" vs "pipelines" repos.

Reply:
Neither of those will get used, I think when we have some more confidence and are ready to run it on all the repos we'll use the pipelines.json.